### PR TITLE
Better Simplified Chinese Localization

### PR DESCRIPTION
**Pull request by nixzhu**
- Improved some translations.
- Added two more strings.

### CODE REVIEW

**Comment by Jerry23011** [comment]
OK, so most fixes are with the punctuations, it won't make any big difference, but thanks for that anyway.

I would personally prefer `邮件` over `电邮` because we know `邮件` means email, and it is what Apple uses for their Mail app. If you really want to emphasize it's "Email", then perhaps try `电子邮件`? `电邮` is rather old-styled  :)

Secondly, `且能鼓励我完善它` doesn't sound like a native Chinese speaker talking to the users, would you please keep the original one, thanks~

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 // MARK: Instances
 "instance.info.domains" = "域名";
-"instance.info.email" = "电邮";
+"instance.info.email" = "邮件";
```
Comment (Jerry23011) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:34]: Thank you so much :)

**Comment by nixzhu** [comment]
@Jerry23011 新年快乐！The email's translation is updated.

**Comment by Dimillian** [comment]
🚀 let's got!

**Pull request merged**